### PR TITLE
Added namespace updates too examples

### PR DESCRIPTION
**Pull request by Klaven**
Per this issue: https://github.com/kubeflow/website/issues/1686 I added updating the namespace too the hyperparameter tuning examples.

First PR, :) so I'm sorry if this is missing more info needed here! please just let me know! It's only WIP because I want to see it's preview first but I think it is all there!

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1813)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1813--competent-brattain-de2d6d.netlify.com/docs/components/hyperparameter-tuning/hyperparameter/

File: content/docs/components/hyperparameter-tuning/hyperparameter.md [hunk 1]
```diff
@@ -290,14 +290,24 @@ View the results of the experiment in the Katib UI:
 Run the following command to launch an experiment using the Kubeflow's
```
Comment (sarahmaddox) [inline content/docs/components/hyperparameter-tuning/hyperparameter.md]: Change "command" to "commands", as there are now multiple commands.

File: content/docs/components/hyperparameter-tuning/hyperparameter.md [hunk 2]
```diff
@@ -308,14 +318,24 @@ Follow the steps as described for the *random algorithm example*
 Run the following command to launch an experiment using Kubeflow's PyTorch
```
Comment (sarahmaddox) [inline content/docs/components/hyperparameter-tuning/hyperparameter.md]: Change "command" to "commands", as there are now multiple commands.

File: content/docs/components/hyperparameter-tuning/hyperparameter.md (hunk 1 above)
Comment (Klaven) [reply content/docs/components/hyperparameter-tuning/hyperparameter.md]: Done! do you want me to squash it?

Comment (sarahmaddox) [reply content/docs/components/hyperparameter-tuning/hyperparameter.md]: Thanks! No squashing needed.

**Pull request merged**